rfc6962-bis.xml

```diff
@@ -1643,13 +1643,7 @@ corrective action when a misissue is detected.
       </section>
       <section title="Redaction of Public Domain Name Labels">
         <t>
-          CAs SHOULD NOT redact domain name labels in precertificates such that the entirety of the domain space below the unredacted part of the domain name is not owned or controlled by a single entity
-(e.g. <spanx style="verb">?.com</spanx> and
-<spanx style="verb">?.co.uk</spanx> would both be problematic). Logs
-MUST NOT reject any precertificate that is overly redacted but which is
-otherwise considered compliant. It is expected that monitors will treat overly
-redacted precertificates as potentially misissued. TLS clients MAY reject a
-certificate whose corresponding precertificate would be overly redacted, perhaps using the same mechanism for determining whether a wildcard in a domain name of a certificate is too broad.
+          Redaction of domain name labels carries the same risks as the use of wildcards (See Section 7.2 of <xref target="RFC6125"/>, for example). CAs should take care not to redact domain name labels in precertificates too broadly, so that the entirety of the domain space below the unredacted part of the domain name may be controlled by multiple entities, such as <xref target="Public.Suffix.List">public suffixes</xref> (e.g. <spanx style="verb">?.com</spanx> and <spanx style="verb">?.co.uk</spanx> would both be problematic). Logs MUST NOT reject any precertificate that is overly redacted but which is otherwise considered compliant. It is expected that monitors will treat overly redacted precertificates as potentially misissued. TLS clients MAY reject a certificate whose corresponding precertificate would be overly redacted, perhaps using the same mechanism for determining whether a wildcard in a domain name of a certificate is too broad.
```

Comment:
I don't think "take care not to redact...*so* that the entirety of the domain space...may be controlled by multiple entities" expresses the intended meaning clearly.  I've had a go at rephrasing this section...

"Redaction of domain name labels carries the same risks as the use of wildcards (see Section 7.2 of &lt;xref target="RFC6125"/&gt;, for example). If the entirety of the domain space below the unredacted part of a domain name is not controlled by a single entity (e.g. &lt;spanx style="verb"&gt;?.com&lt;/spanx&gt;, &lt;spanx style="verb"&gt;?.co.uk&lt;/spanx&gt; and other &lt;xref target="Public.Suffix.List"&gt;public suffixes&lt;/xref&gt;), then the domain name may be considered by clients to be overly redacted.
</t>
<t>
CAs should take care to avoid overly redacting domain names in precertificates. It is expected that monitors will treat precertificates that contain overly redacted domain names as potentially misissued. TLS clients MAY reject a certificate if the &lt;xref target="reconstructing_tbscertificate"&gt;reconstructed TBSCertificate&lt;/xref&gt; contains any overly redacted domain names.whose corresponding precertificate would be overly redacted."

Note that I dropped the "perhaps using the same mechanism" clause.  This seems redundant now, given that we're citing the PSL as one way of doing it.